pytest_selenium/pytest_selenium.py

```diff
@@ -48,11 +55,20 @@ def capabilities(request, driver_class, chrome_options, firefox_options,
     """Returns combined capabilities"""
     capabilities = copy.deepcopy(session_capabilities)  # make a copy
     if driver_class == webdriver.Remote:
-        browser_name = str(capabilities.get('browserName')).lower()
-        if browser_name == 'chrome':
-            capabilities.update(chrome_options.to_capabilities())
-        elif browser_name == 'firefox':
-            capabilities.update(firefox_options.to_capabilities())
+        browser = capabilities.get('browserName', '').upper()
+        key, options = (None, None)
+        if browser == 'CHROME':
```

Comment:
```        if browser == 'CHROME':
            key = 'chromeOptions'
            options = chrome_options.to_capabilities()
            if 'chromeOptions' not in options:
                key = 'goog:chromeOptions'
        elif browser == 'FIREFOX':```

Reply:
Good suggestion, though I tweaked it a little further.

Reply:
@davehunt But isn't `goog:chromeOptions` the key that is being replaced by just `chromeOptions`?